dist/js/medium-editor.js

```diff
@@ -4267,11 +4408,11 @@ function MediumEditor(elements, options) {
         },
 
         cleanPaste: function (text) {
-            this.pasteHandler.cleanPaste(text);
+            this.getExtensionByName('paste').cleanPaste(text);
         },
 
         pasteHTML: function (html, options) {
-            this.pasteHandler.pasteHTML(html, options);
+            this.getExtensionByName('paste').pasteHTML(html, options);
```

Comment:
i like this. in my initial version I had all the extensions register themselves on the editor instance (eg: passing a `paste` extension would override the `this.paste` value ... then code could defensively be written as

```
if(this.paste){ this.paste.pasteHTML(html, options) }
```

your above pasteHTML call will throw if someone turned off paste with `paste: false` option, no?